### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by oliverviljamaa**
https://transferwise.github.io/public-navigation/branch/i18n

# v0.6.0
## Adds translations

Translation bundles built with [`webpack-translations-plugin`](https://www.npmjs.com/package/webpack-translations-plugin)
are now exposed in the package.

The development environment and the docs use the source messages.

### CODE REVIEW

**Comment by oliverviljamaa** [comment]
@Tankenstein Considered it, but wasn't sure, I think `text` is more obvious as `link text` seems to be more popular than `link label`.

**Comment by Tankenstein** [comment]
I can see that, but it isn‘t consistent with our standard components

File: src/i18n/translate.spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable global-require */
+
+describe('Translate', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  function getTranslateForTranslations(translations) {
+    jest.doMock('translations', () => translations, { virtual: true });
+    return require('./translate').default;
```
Comment (codewhisperer) [inline src/i18n/translate.spec.js]: can you not import this up top?

Comment (oliverviljamaa) [reply src/i18n/translate.spec.js]: Nah, I don't want it hoisted to the top, hence `doMock`. Also, I want to be able to change what the mock returns between tests.

File: src/items/items.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { translate } from '../i18n';
+
+export function getItemsInLanguage(language) {
+  return items.map(item => {
+    const translatedItem = translateItem(item, language);
+
+    if (item.items) {
+      return {
+        items: item.items.map(subItem => translateItem(subItem, language)),
+        ...translatedItem,
```
Comment (codewhisperer) [inline src/items/items.js]: 😍

File: src/i18n/translate.spec.js (hunk 1 above)
Comment (codewhisperer) [reply src/i18n/translate.spec.js]: Ok, but doesn't it always return the same imported thing?

Comment (oliverviljamaa) [reply src/i18n/translate.spec.js]: Nah, the function will be produce different outputs based on what the `translations` module in `./translate` resolves to.

Comment (codewhisperer) [reply src/i18n/translate.spec.js]: Ah cool 👍

**Pull request merged**